cypress/component/DataTable.cy.tsx

```diff
@@ -0,0 +1,70 @@
+import DataTable from '../../src/components/DataTable';
+
+const exampleFileName = 'ds003653_participant.tsv';
+const exampleFilePath = `examples/${exampleFileName}`;
+
+describe('DataTable', () => {
+  it('should render the component correctly', () => {
+    cy.mount(<DataTable />);
+    cy.get('[data-cy="data-table-card"]').should('be.visible');
+    cy.get('[data-cy="data-table-card"]').should(
+      'contain',
+      'Upload your tabular phenotypic file (.tsv format)'
+    );
+  });
+  beforeEach(() => {
+    cy.mount(<DataTable />);
+
+    // Load the file from the fixtures folder
+    cy.fixture(exampleFilePath, 'binary').then((fileContent) => {
+      // Convert the binary content to a Blob
+      const blob = Cypress.Blob.binaryStringToBlob(fileContent, 'text/tab-separated-values');
+
+      // Create a File object from the Blob
+      const testFile = new File([blob], exampleFileName, { type: 'text/tab-separated-values' });
+
+      // Simulate clicking the upload area
+      cy.get('[data-cy="upload-area"]').click();
+
+      // Trigger the file upload by selecting the file
+      cy.get('input[type="file"]').then((input) => {
+        // Cast input[0] to HTMLInputElement to access the files property
+        const fileInput = input[0] as HTMLInputElement;
+
+        // Create a DataTransfer object to simulate the file drop
+        const dataTransfer = new DataTransfer();
+        dataTransfer.items.add(testFile);
+        fileInput.files = dataTransfer.files;
+
+        // Trigger the change event manually
```

Comment:
This works, but like I said on the mocks.ts file, it'd be very nice if we could just provide the file object to a store method to set the store without having to go through the UI. I don't think that'll be a big change now since there isn't much connected to the store yet